packages/cli/templates/Update.ts

```diff
@@ -0,0 +1,139 @@
+import { App, FS_TOKEN, IFileSystem, PackageManager, ProjectConfig, PackageDefinition, Util } from "@igniteui/cli-core";
+import * as path from "path";
+import { getReactUpgradeablePackages } from "./react/package-resolve";
+import { getWebcomponentsUpgradeablePackages } from "./webcomponents/package-resolve";
+
+export async function updateWorkspace(rootPath: string): Promise<boolean> {
+	let guideLink = "";
+	let logicFilesExtension = "";
+	let upgradeable: PackageDefinition[] = [];
+
+	const config = ProjectConfig.getConfig();
+	const framework = config.project.framework;
+	switch (framework.toLowerCase()) {
+		// case "angular":
+		// 	guideLink =
+		// 	  "https://www.infragistics.com/products/ignite-ui-angular/angular/components/general/ignite-ui-licensing";
+		// 	break;
```

Comment:
Do we need this commented out chunk?

Reply:
Depends if we want to keep refactoring and support Angular upgrade here in the same file.